src/pages/index.js

```diff
@@ -4,13 +4,16 @@ import "../pages/index.css";
 import PopupWithImage from "../components/PopupWithImage.js";
 import PopupWithForm from "../components/PopupWithForm.js";
 import UserInfo from "../components/UserInfo.js";
-import { initialCards } from "../utils/constants.js";
+// get rid of this array
+// import { initialCards } from "../utils/constants.js";
 import Section from "../components/Section.js";
+import Api from "../components/Api.js";
+import PopupDeleteCard from "../components/PopupDeleteCard.js";
 
 const config = {
```

Comment:
From the 8th sprint all constants (including `initialCards` and `config`) should be in file `constants.js` in folder `utils` to clean up the main file.